src/app/zap-templates/zcl/data-model/chip/joint-fabric-administrator-cluster.xml

```diff
@@ -0,0 +1,94 @@
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+
+  <enum name="ICACResponseStatusEnum" type="enum8">
+    <cluster code="0x0753"/>
+    <item name="OK" value="0x00"/>
+    <item name="InvalidPublicKey" value="0x01"/>
+    <item name="InvalidICAC" value="0x02"/>
+  </enum>
+
+  <enum name="TransferAnchorResponseStatusEnum" type="enum8">
+    <cluster code="0x0753"/>
+    <item name="OK" value="0x00"/>
+    <item name="TransferAnchorStatusDatastoreBusy" value="0x01"/>
+    <item name="TransferAnchorStatusNoUserConsent" value="0x02"/>
+  </enum>
+
+  <cluster>
+    <domain>General</domain>
+    <name>Joint Fabric Administrator</name>
+    <code>0x0753</code>
+    <define>JOINT_FABRIC_ADMINISTRATOR_CLUSTER</define>
+    <description>Joint Fabric Administrator Cluster.</description>
+
+    <attribute side="server" code="0x0000" name="ADMINISTRATOR_FABRIC_INDEX" define="ADMINISTRATOR_FABRIC_INDEX" type="fabric_idx" writable="true" default="0">
+      <description>AdministratorFabricIndex</description>
+      <access op="read" privilege="administer"/>
+      <access op="write" privilege="administer"/>
+      <mandatoryConform/>
+    </attribute>
+
+    <command source="client" code="0x00" name="ICACCSRRequest" response="ICACCSRResponse" optional="false">
+      <description>ICACCSRRequest</description>
+      <access op="invoke" privilege="administer"/>
+    </command>
+
+    <command source="server" code="0x01" name="ICACCSRResponse" optional="false">
+      <description>ICACCSRResponse</description>
+      <arg name="ICACCSR" type="octet_string" length="400"/>
+    </command>
+
+    <command source="client" code="0x02" name="AddICAC" response="ICACResponse" optional="false">
+      <description>AddICAC</description>
+      <arg name="ICACValue" type="octet_string" length="400"/>
+      <access op="invoke" privilege="administer"/>
+    </command>
+
+    <command source="server" code="0x03" name="ICACResponse" optional="false">
+      <description>ICACResponse</description>
+      <arg name="StatusCode" type="ICACResponseStatusEnum"/>
+    </command>
+
+    <command source="client" code="0x04" name="OpenJointCommissioningWindow" optional="false">
+      <description>OpenJointCommissioningWindow</description>
+      <arg name="CommissioningTimeout" type="int16u"/>
+      <arg name="PAKEPasscodeVerifier" type="octet_string"/>
+      <arg name="Discriminator" type="int16u"/>
+      <arg name="Iterations" type="int32u"/>
+      <arg name="Salt" type="octet_string" length="32"/>
+      <access op="invoke" privilege="administer"/>
+    </command>
+
+    <command source="client" code="0x05" name="TransferAnchorRequest" response="TransferAnchorResponse" optional="false">
```

Comment:
Given that Anchor Transfer functionality is postponed for Matter 1.6, I think all the Anchor Transfer commands and related enums can be removed from this PR.

Reply:
Why not make them provisional in 1.5?  We can then implement them in 1.6.

Reply:
Yes, those commands can be kept as provisional for now in the spec. But in the core SDK the suggestion would be to remove them for now to avoid creating confusion - I am thinking that we'll refactor those commands a lot once we discuss the anchor transfer functionality in detail.

Reply:
Removed in the latest commit.